### PR TITLE
F0 implementing trainer add pages methods

### PR DESCRIPTION
**Pull request by GolamCh**

### CODE REVIEW

**Review by jackingham — APPROVED** [review]
Nice work

File: src/test/java/com/sparta/eng82/interfaces/pages/navpages/trainer/addpages/AddGroupPage.java [hunk 1]
```diff
@@ -1,22 +1,23 @@
 package com.sparta.eng82.interfaces.pages.navpages.trainer.addpages;
 
+import com.sparta.eng82.components.pages.navpages.trainer.addpages.AddGroupPageImpl;
 import com.sparta.eng82.interfaces.pages.navpages.trainer.ManageGroupPage;
 
 public interface AddGroupPage {
 
-    AddGroupPage selectStream(String streamName);
```
Comment (alexsikorski) [inline src/test/java/com/sparta/eng82/interfaces/pages/navpages/trainer/addpages/AddGroupPage.java:7]: Should be AddGroupPage not implementation

**Review by alexsikorski — CHANGES_REQUESTED** [review]
Interface return types changed to Impl, please fix

**Review by AJMalcolm — APPROVED** [review]
Performed the requested changes

**Review by alexsikorski — APPROVED** [review]
Thanks!

**Pull request merged**